examples/mercari.py

```diff
@@ -1,9 +1,174 @@
+import argparse
 import os.path as osp
+from typing import List
 
+import torch
+import torch.nn.functional as F
+from sentence_transformers import SentenceTransformer
+from torch import Tensor
+from tqdm import tqdm
+
+from torch_frame import stype
+from torch_frame.config.text_embedder import TextEmbedderConfig
+from torch_frame.data import DataLoader
 from torch_frame.datasets import Mercari
+from torch_frame.nn import (
+    EmbeddingEncoder,
+    FTTransformer,
+    LinearEmbeddingEncoder,
+    LinearEncoder,
+)
+from torch_frame.nn.encoder.stype_encoder import \
+    MultiCategoricalEmbeddingEncoder  # noqa
+
+
+class PretrainedTextEncoder:
+    def __init__(self, device: torch.device):
```

Comment:
nit: https://peps.python.org/pep-0484/#the-meaning-of-annotations
```suggestion
    def __init__(self, device: torch.device) -> None:
```